src/desktop/apps/home/stylesheets/featured_items.styl

```diff
@@ -1,3 +1,5 @@
+@import '../../../lib/palette'
```

Comment:
Ah, mostly I forgot there's a difference (it's been a *while*). **However** — since there's only mixins/functions/variables in this file, it shouldn't actually make a difference.